apollo-federation/src/query_graph/graph_path.rs

```diff
@@ -2338,19 +2338,12 @@ where
                         write!(f, "{node}")
                     } else {
                         let edge = &graph[e];
-                        let label = edge.transition.to_string();
-
-                        if let Some(conditions) = &edge.conditions {
-                            write!(f, " --[{conditions} ⊢ {label}]--> {node}")
-                        } else if let Some(conditions) = &edge.override_condition {
-                            write!(f, " --[{conditions} ⊢ {label}]--> {node}")
-                        } else if !matches!(
-                            edge.transition,
-                            QueryGraphEdgeTransition::SubgraphEnteringTransition
-                        ) {
-                            write!(f, " --[{label}]--> {node}")
+                        let label = edge.to_string();
```

Comment:
I want to make sure I understand this. The label now contains the logic for all but the final `else` statement of the old if-else chain, ya?